common/src/main/java/org/opensearch/ml/common/connector/MLPreProcessFunction.java

```diff
@@ -22,6 +23,7 @@ public class MLPreProcessFunction {
     public static final String TEXT_DOCS_TO_COHERE_EMBEDDING_INPUT = "connector.pre_process.cohere.embedding";
     public static final String TEXT_DOCS_TO_OPENAI_EMBEDDING_INPUT = "connector.pre_process.openai.embedding";
     public static final String TEXT_DOCS_TO_BEDROCK_EMBEDDING_INPUT = "connector.pre_process.bedrock.embedding";
+    public static final String TEXT_IMAGE_TO_BEDROCK_EMBEDDING_INPUT = "connector.pre_process.multimodal.embedding";
```

Comment:
How about adding bedrock to the name "connector.pre_process.bedrock.multimodal_embedding" ? That can keep consistent with `"connector.pre_process.bedrock.embedding"`